src/modules/feedback/components/FeedbackBox.js

```diff
@@ -36,16 +36,26 @@ type PropsType = {|
   comment: string,
   onCommentChanged: SyntheticInputEvent<HTMLTextAreaElement> => void,
   onFeedbackOptionChanged: FeedbackVariant => void,
-  onSubmit: () => void,
+  onSubmit: (value?: string) => void,
   t: TFunction,
   closeFeedbackModal: () => void,
-  theme: ThemeType
+  theme: ThemeType,
+  feedbackSent: string
```

Comment:
You should use the right type here (`SendingStatusType`).

Reply:
Type changed